<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetRightView</class>
 <widget class="QWidget" name="widgetRightView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widgetRightView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="BDLS.qrc">
     <normaloff>:/BDLS/icons/zoom-in-32.png</normaloff>:/BDLS/icons/zoom-in-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_In</string>
   </property>
   <property name="toolTip">
    <string>Zoom_In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="BDLS.qrc">
     <normaloff>:/BDLS/icons/zoom-out-32.png</normaloff>:/BDLS/icons/zoom-out-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom_Out</string>
   </property>
  </action>
  <action name="actionContinuous">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Continuous</string>
   </property>
   <property name="toolTip">
    <string>Continuous</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BDLS.qrc"/>
 </resources>
 <connections/>
</ui>
